conf/messages.en

```diff
@@ -1061,19 +1061,20 @@ uploadInProgress.button = Check progress
 haveYouReceivedADecision.heading = Previous rulings based on similar methods
 haveYouReceivedADecision.para.1 = This must be a ruling decision from Great Britain (England, Scotland and Wales). It does not need to be related to your current application.
 haveYouReceivedADecision.para.2 = Telling us about similar rulings you have received will help us process your current application.
-haveYouReceivedADecision.error.required = Tell us if you have ever received a ruling based on a similar method
+haveYouReceivedADecision.error.required = Select yes if you have received a previous ruling based on a similar method
 haveYouReceivedADecision.legend = Have you ever received a ruling based on a similar method?
 haveYouReceivedADecision.checkYourAnswersLabel = Have you ever received a ruling based on a similar method?
 haveYouReceivedADecision.change.hidden = whether you have ever received a ruling based on a similar method
 
+tellUsAboutYourRuling.title = About the previous ruling
```

Comment:
My understanding is that this should match the question you added as a label?

_"Page title on page 3 is updated in line with the changed question"_

Reply:
yeah my bad your right will update

Reply:
Updated

<img width="1275" alt="Screenshot 2024-01-12 at 13 45 54" src="https://github.com/hmrc/advance-valuation-rulings-frontend/assets/47248104/980091db-d4b5-4a40-af9d-4429962e64d2">

Reply:
No this is the heading, the title is on the tab.
The "TO BE - Page 3.png" screenshots on the ticket has "About the previous ruling" for the H1. You added a key for the title, and looks like the ask is to match the tab text to be "Can you provide more information about the previous ruling?".

Reply:
ah okay keep the original h1 but change the tab title.

Reply:
weird thing is the title almost always matches the h1.

Reply:
Confirmed with design